spk/salt-master/Makefile

```diff
@@ -1,38 +1,54 @@
 SPK_NAME = salt-master
-SPK_VERS = 2015.5.11
-SPK_REV = 9
+SPK_VERS = 3003.3
+SPK_REV  = 1
 SPK_ICON = src/salt-master.png
 
-DEPENDS =
+SPK_DEPENDS = "python38"
+
+DEPENDS = cross/patch
+
+BUILD_DEPENDS  = cross/python38 cross/setuptools cross/pip
+BUILD_DEPENDS += cross/wheel
+BUILD_DEPENDS += cross/salt
+
+include ../../mk/spksrc.archs.mk
+
+# Requirements file generation
+# /usr/local/python3/bin/python3 -mvirtualenv --python=python3  salt-env
+# virtualenv --python=python3  salt-env
+# source salt-env/bin/activate
+# pip install salt
+# pip freeze > requirements.txt
 WHEELS = src/requirements.txt
-SPK_DEPENDS = "python>2.7.8-10"
 
-MAINTAINER = SynoCommunity
-DESCRIPTION = Salt, a new approach to infrastructure management, is easy enough to get running in minutes, scalable enough to manage tens of thousands of servers, and fast enough to communicate with those servers in seconds.
-RELOAD_UI = yes
+MAINTAINER   = SynoCommunity
+DESCRIPTION  = Salt, a new approach to infrastructure management, is easy enough to get running in minutes, scalable enough to manage tens of thousands of servers, and fast enough to communicate with those servers in seconds.
+RELOAD_UI    = yes
 DISPLAY_NAME = Salt Master
-CHANGELOG = "1. Update to 2015.5.11<br/>2. Fix salt-api<br/>3. Update dependencies"
+CHANGELOG    = "Update to 3003.3"
 
-HOMEPAGE   = https://www.saltstack.com/
-LICENSE    = Apache v2
-
-INSTALLER_SCRIPT = src/installer.sh
-SSS_SCRIPT       = src/dsm-control.sh
-FWPORTS          = src/${SPK_NAME}.sc
+HOMEPAGE     = https://www.saltstack.com/
+LICENSE      = Apache
+LICENSE_FILE = src/LICENSE
 
 POST_STRIP_TARGET = salt-master_extra_install
 
-# Pure Python package, make sure ARCH is not defined
-override ARCH=
+# Service configuration
+CONF_DIR = src/conf
+SERVICE_SETUP = src/service-setup.sh
+STARTABLE = yes
+
+SPK_USR_LOCAL_LINKS = bin:env/bin/salt
+SPK_USR_LOCAL_LINKS = bin:env/bin/salt-cp
+SPK_USR_LOCAL_LINKS = bin:env/bin/salt-key
+SPK_USR_LOCAL_LINKS = bin:env/bin/salt-master
+SPK_USR_LOCAL_LINKS = bin:env/bin/salt-run
+SPK_USR_LOCAL_LINKS = bin:env/bin/salt-unity
+SPK_USR_LOCAL_LINKS = bin:env/bin/spm
 
 include ../../mk/spksrc.spk.mk
 
 .PHONY: salt-master_extra_install
 salt-master_extra_install:
-	install -m 755 -d $(STAGING_DIR)/var
-	install -m 755 -d $(STAGING_DIR)/etc
-	install -m 644 src/master $(STAGING_DIR)/etc/master
-	install -m 755 -d $(STAGING_DIR)/share/extensions
-	install -m 755 -d $(STAGING_DIR)/share/extensions/auth
-	install -m 644 src/synology.py $(STAGING_DIR)/share/extensions/auth/synology.py
-
+	install -m 644 src/rsax931.py.patch $(STAGING_DIR)/share
```

Comment:
Please forgive this question by why can't the patch applied like the other patches?

Reply:
Dont shoot the messenger :D I have just copied the previous installation.

I can try and see if I can figure out how to patch a wheel package ( ./cross/salt ).

Reply:
Haha, don't worry. As a reviewer I just have to look for unusual things and this is a little unusual IHMO. 
Thanks :), but I also think it's not really worth sinking a lot of time into it either. So don't worry about this one.

Reply:
patching moved to cross/salt wheel package.